user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQIOnInit.md

```diff
@@ -1,11 +1,25 @@
 ---
-uid: IGQIOnInit
+uid: GQI_IGQIOnInit
 ---
 
-# IGQIOnInit
+# IGQIOnInit Interface
 
-The *IGQIOnInit* interface is called when the data source is initialized, for example when the data source is selected in the query builder or when a dashboard using a query with ad hoc data is opened. It can for instance be used to connect to a database. This interface has one method:
+## Definition
 
-| Method | Input arguments | Output arguments | Description |
-|--|--|--|--|
-| OnInit | OnInitInputArgs | OnInitOutputArgs | Indicates that an instance of the class has been created. |
+Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
+Assembly: `SLAnalyticsTypes.dll`
+
+The *IGQIOnInit* interface is called when the data source is initialized, for example when the data source is selected in the query builder or when a dashboard using a query with ad hoc data is opened. It can for instance be used to connect to a database.
```

Comment:
The interface itself can't be called.
Similar to the OnDestroy, we should probably have an exhaustive list of the possible scenarios in which the OnInit method can be called.